src/components/modals/signupModals/signupCollaborator2.tsx

```diff
@@ -77,6 +130,26 @@ export default function SignupModalCollaborator2() {
             sign up to be a beta tester
           </label>
         </div>
+        <div className="ReferralPartnerButton mt-0 flex items-center gap-2">
+          <SiteButton
+            variant="hollow"
+            colorScheme="b4"
+            aria="referralPartner"
+            size="smallCircle"
+            isSelected={watch("referralPartner")}
+            onClick={() => {
+              const newValue = !watch("referralPartner");
```

Comment:
Since this is used twice, and also because it's a bit lengthy to be declared in the onclick, it would be better to create a function that takes "betaTester" or "referralPartner" as an input and then runs these functions for here and the button above.